.gitmodules

```diff
@@ -12,3 +12,7 @@
 	path = submodules/xdp-for-windows
 	url = https://github.com/microsoft/xdp-for-windows.git
 	branch = release/1.1
+[submodule "submodules/openssl"]
+	path = submodules/openssl
+	url = https://github.com/nhorman/openssl.git
+	branch = quic-secret-yielding-fix
```

Comment:
Just putting a TODO here to remember to change this back to the official release branch once the fix is available there.